### PR TITLE
[Ide] Re-add old text log viewer

### PR DESCRIPTION
**Pull request by rodrmoya**

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs [hunk 1]
```diff
@@ -196,17 +201,26 @@ protected override void Initialize (IPadWindow window)
...
 			toolbar.Add (sep);
 
-			logBtn = MakeButton ("md-message-log", "toggleBuildOutput", out logBtnLbl);
+			logBtn = MakeButton ("md-message-log", "toggleBuildOutput", false, out logBtnLbl);
 			logBtn.Accessible.Name = "ErrorPad.LogButton";
-			logBtn.TooltipText = GettextCatalog.GetString ("Show build output");
-			logBtn.Accessible.Description = GettextCatalog.GetString ("Show build output");
-
-			logBtnLbl.Text = GettextCatalog.GetString ("Build Output");
+			logBtn.TooltipText = GettextCatalog.GetString ("Text build output");
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: Let's use just "Build Output" for the old option.

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs [hunk 2]
```diff
@@ -196,17 +201,26 @@ protected override void Initialize (IPadWindow window)
...
+			logBtnLbl.Text = GettextCatalog.GetString ("Text Build Output");
 			logBtn.Accessible.SetTitle (logBtnLbl.Text);
-
-			logBtn.Clicked += HandleLogBtnClicked;
+			logBtn.Toggled += HandleTextLogToggled;
 			toolbar.Add (logBtn);
 
+			buildLogBtn = MakeButton ("md-message-log", "toggleBuildOutput", out logBtnLbl);
+			buildLogBtn.Accessible.Name = "ErrorPad.BuildLogButton";
+			buildLogBtn.TooltipText = GettextCatalog.GetString ("Structured build output");
```
Comment (slluis) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: Should be "Structured **B**uild **O**utput"

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs (hunk 1 above)
Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: fixed

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs (hunk 2 above)
Comment (rodrmoya) [reply main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: fixed

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs [hunk 3]
```diff
@@ -196,17 +201,26 @@ protected override void Initialize (IPadWindow window)
...
 
+			buildLogBtn = MakeButton ("md-message-log", "toggleBuildOutput", out logBtnLbl);
+			buildLogBtn.Accessible.Name = "ErrorPad.BuildLogButton";
+			buildLogBtn.TooltipText = GettextCatalog.GetString ("Structured Build Output");
+			buildLogBtn.Accessible.Description = GettextCatalog.GetString ("Structured Build Output");
+
+			logBtnLbl.Text = GettextCatalog.GetString ("Structured Build Output");
+			buildLogBtn.Accessible.SetTitle (logBtnLbl.Text);
+
+			buildLogBtn.Clicked += HandleBinLogClicked;
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Gui.Pads/ErrorListPad.cs]: In example, leak here.

**Comment by slluis** [comment]
@Therzok ok now?

**Comment by slluis** [comment]
@monojenkins backport release-7.6

**Pull request merged**